framework/custom_assertions.py

```diff
@@ -1,16 +1,5 @@
-"""Custom Assertions Module
-
-This module provides a class `CustomAssertions` that provides custom assertions
-for the `Element` class.
-
-The `CustomAssertions` class provides a way to write custom assertions for the
-`Element` class. It allows you to write assertions that are not already provided
-by the `Element` class.
-
-The `CustomAssertions` class provides the following methods:
-
-- `to_have`: Asserts that the element has a certain text or attribute.
-- `to_match`: Asserts that the element's attribute matches a certain regex pattern.
+"""
```

Comment:
In previously merged PR we added above docstrings. Is it intentional to remove them?